lib/ansible/modules/find.py

```diff
@@ -460,21 +460,22 @@ def main():
                         continue
 
                     r = {'path': fsname}
-                    if params['file_type'] == 'any':
+                    file_type = to_text(params['file_type'], errors='surrogate_or_strict')
```

Comment:
unrelated changes in this file

Reply:
just testing since PR is 'reproducer', but it seems to no t be the issue